lib/LaTeXML/Package/thmtools.sty.ltxml

```diff
@@ -62,7 +62,7 @@ sub thmtools_style {
   if (my $bodyfont = $kv && $kv->getValue('bodyfont')) {
     $parameters{'\thm@bodyfont'} = $bodyfont; }
   if (my $headformat = ToString(($kv && $kv->getValue('headformat')) || '')) {
-    $parameters{'\thm@swap'} = $headformat eq 'swapnumber'; }
+    $parameters{'thm@swap'} = $headformat eq 'swapnumber'; }
```

Comment:
I'm not closely familiar with the code here, but if `thm@swap` loses the backslash, shouldn't all other `thm@` parameters lose it as well? As in `\thm@bodyfont` right above? 

It's a little unclear if they're subtly different.

Reply:
Yeah, way too subtly different. The others are registers, and really should be. The swap is effectively boolean, though it could be forced to be a register.